spec/features/map/nlp_spatial_temporal_extraction_spec.rb

```diff
@@ -40,11 +40,11 @@
     end
 
     it 'filters collection results' do
-      expect(page).to have_content('6325 Matching Collections')
+      expect(page).to have_content('11553 Matching Collections')
```

Comment:
This file sticks out in the review. I don't think you shouldn't have changed anything that caused this test to need to be updated. I'm also unsure of why the collections_requests.yml and collections_responses.yml files were changed